### PR TITLE
Add Multi-Selector

### PR DESCRIPTION
**Pull request by lungben**
Add a Multi-Selector (using html `select multiple`).

### CODE REVIEW

File: src/Builtins.jl [hunk 1]
```diff
@@ -170,19 +171,52 @@ function show(io::IO, ::MIME"text/html", select::Select)
...
+`@bind veg MultiSelect(["potato" => "🥔", "carrot" => "🥕"], default=["carrot"])`"""
+struct MultiSelect <: AbstractSelect
+    options::Array{Pair{<:AbstractString,<:Any},1}
+    default::Union{Missing, AbstractVector{AbstractString}}
+end
+MultiSelect(options::Array{<:AbstractString,1}; default=missing) = MultiSelect([o => o for o in options], default)
+MultiSelect(options::Array{<:Pair{<:AbstractString,<:Any},1}; default=missing) = MultiSelect(options, default)
+
+function show(io::IO, ::MIME"text/html", select::MultiSelect)
+    withtag(io, Symbol("select multiple")) do
```
Comment (fonsp) [inline src/Builtins.jl:207]: woa this is well found!

File: src/Builtins.jl [hunk 2]
```diff
@@ -170,19 +171,52 @@ function show(io::IO, ::MIME"text/html", select::Select)
...
+                show(io, MIME"text/html"(), o.second)
+            else
+                print(io, o.second)
+            end
+            print(io, "</option>")
+        end
+    end
+end
+
+get(select::AbstractSelect) = ismissing(select.default) ? first(select.options).first : select.default
```
Comment (fonsp) [inline src/Builtins.jl:184]: this is not the default value for MultiSelect, so let's just make them two separate types without a supertype?

The default value should be 
```julia
get(select::MultiSelect) = ismissing(select.default) ? Any[] : select.default
```

also see my comment about the `default`

File: src/Builtins.jl [hunk 3]
```diff
@@ -170,19 +171,52 @@ function show(io::IO, ::MIME"text/html", select::Select)
...
+
+See the [Mozilla docs about `select`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select)
+
+# Examples
+`@bind veg MultiSelect(["potato", "carrot"])`
+
+`@bind veg MultiSelect(["potato" => "🥔", "carrot" => "🥕"])`
+
+`@bind veg MultiSelect(["potato" => "🥔", "carrot" => "🥕"], default=["carrot"])`"""
+struct MultiSelect <: AbstractSelect
```
Comment (fonsp) [inline src/Builtins.jl:199]: Did you name it after the [REPL menu](https://docs.julialang.org/en/v1/stdlib/REPL/#MultiSelectMenu-1)? Nice!

File: src/Builtins.jl [hunk 4]
```diff
@@ -170,19 +171,52 @@ function show(io::IO, ::MIME"text/html", select::Select)
...
+            end
+            print(io, "</option>")
+        end
+    end
+end
+
+get(select::AbstractSelect) = ismissing(select.default) ? first(select.options).first : select.default
+
+
+"""A multi-selector (`<select multi>`) - the user can choose one or more of the `options`, an array of `Strings.
```
Comment (fonsp) [inline src/Builtins.jl:187]: add:
```
See [`Select`](@ref) for a version that allows only one selected item.

File: src/Builtins.jl [hunk 5]
```diff
@@ -146,6 +146,7 @@ end
 
 get(textfield::TextField) = textfield.default
 
+abstract type AbstractSelect end
 
 """A dropdown menu (`<select>`) - the user can choose one of the `options`, an array of `String`s.
```
Comment (fonsp) [inline src/Builtins.jl:151]: add:
```
See [`MultiSelect`](@ref) for a version that allows multiple selected items.

File: src/Builtins.jl (hunk 2 above)
Comment (lungben) [reply src/Builtins.jl:184]: You are right, I'll correct this.

File: src/Builtins.jl (hunk 4 above)
Comment (lungben) [reply src/Builtins.jl:187]: done

File: src/Builtins.jl (hunk 5 above)
Comment (lungben) [reply src/Builtins.jl:149]: done

**Comment by fonsp** [comment]
Yay thanks!

**Comment by fonsp** [comment]
It's released in `v0.6.0`

**Pull request merged**